python_apps/airtime_analyzer/tests/cuepoint_analyzer_test.py

```diff
@@ -1,3 +1,4 @@
+import lsb_release
```

Comment:
Module not found error. Think uou need python3- packages and added to requirements-dev.txt

Reply:
Or use platform https://docs.python.org/3/library/platform.html#platform.uname

Reply:
I'll use `distro`, I really didn't think this though before using it. `lsb_release` is a bad idea.